Wire-iOS/Sources/UserInterface/Conversation/InputBar/AudioRecordViewController.swift

```diff
@@ -52,9 +51,8 @@ private let zmLog = ZMSLog(tag: "UI")
     let topContainerView = UIView()
     let cancelButton = IconButton()
     let recordingDotView = RecordingDotView()
-    var recordingDotViewVisible: ConstraintGroup?
-    var recordingDotViewHidden: ConstraintGroup?
-    
+    var recordingDotViewVisible = [NSLayoutConstraint]()
+    var recordingDotViewHidden = [NSLayoutConstraint]()
```

Comment:
Using the array literal is usually more idiomatic in Swift.

```suggestion
    var recordingDotViewHidden: [NSLayoutConstraint] = []
```